resource_calendar_rrule/models/resource_calendar.py

```diff
@@ -0,0 +1,275 @@
+                    'afternoon': 4,
+                },
+            ])
+        return result
+
+    @api.multi
+    def _compute_simplified_attendance(self):
+        for this in self:
+            for attendance in this.attendance_ids:
+                this.simplified_attendance = this\
+                    ._simplified_from_attendances()
+
+    @api.model
+    @api.depends('attendance_ids')
+    def _simplified_from_attendances(self):
+        self.ensure_one()
+        result = {'type': 'all', 'data': [], 'data_odd': []}
+        for day in [0, 1, 2, 3, 4, 5, 6]:
+            result['data'].append({
+                'day': day,
+                'morning': 0,
+                'afternoon': 0,
+            })
+            result['data_odd'].append({
+                'day': day,
+                'morning': 0,
+                'afternoon': 0,
+            })
+        warn = ''
+        start = datetime.datetime.today().replace(tzinfo=pytz.utc)
+        stop = False
+        for attendance in self.attendance_ids:
+            for rule in attendance.rrule._rrule:
+                if rule._byeaster or rule._bymonth or rule._bymonthday or\
+                        rule._byyearday or rule._interval not in [1, 2] or\
+                        rule._freq != WEEKLY or not rule._dtstart:
+                    # TODO: this neds to warn somehow if the rrule can't be
+                    # simplified
+                    warn += _('Ignoring %s') % rule
+                    continue
+
+                if rule._dtstart < start:
+                    start = rule._dtstart
+                if rule._until and (not stop or stop > rule._until):
+                    stop = rule._until
+
+                odd = all(
+                    bool(rule._dtstart.isocalendar()[1] % 2)
+                    for occurrence in rule[:2]
+                )
+                key = float(self.env['ir.config_parameter'].get_param(
+                    'resource_calendar_rrule.simplified_afternoon',
+                    DEFAULT_AFTERNOON_HOUR,
+                )) > attendance.hour_from and 'morning' or 'afternoon'
+
+                for day in result['data%s' % (odd and '_odd' or '')]:
+                    if day['day'] not in rule._byweekday:
+                        continue
+                    day[key] += attendance.hour_to - attendance.hour_from
+            for exdate in attendance.rrule._exdate:
+                result.setdefault('exdate', [])
+                result['exdate'].append(fields.Date.to_string(exdate))
+            for rdate in attendance.rrule._rdate:
+                result.setdefault('rdate', [])
+                result['rdate'].append(fields.Date.to_string(rdate))
+
+        def sum_all(x):
+            return x['morning'] + x['afternoon']
+
+        if not sum(map(sum_all, result['data'])):
+            result['data'] = result['data_odd']
+            result.pop('data_odd')
+        elif not sum(map(sum_all, result['data_odd'])):
+            result.pop('data_odd')
```

Comment:
This code is also questionable to me - what if people want to specify an "even" or "odd" schedule with 0 hours in the other week?